packages/frontend/tests/unit/service/current-user-test.ts

```diff
@@ -1,9 +1,26 @@
 import { module, skip } from 'qunit';
-import { setupTest } from 'ember-qunit';
+import { setupTest, test } from 'ember-qunit';
+import CurrentUserService from 'emberclear/services/current-user';
+import { getService } from 'emberclear/tests/helpers';
+import { generateAsymmetricKeys } from 'emberclear/workers/crypto/utils/nacl';
 
 module('Unit | Service | identity', function (hooks) {
   setupTest(hooks);
 
+  let service: CurrentUserService;
+
+  hooks.beforeEach(() => {
+    service = getService('current-user');
+  });
+
+  test('importFromKey where privateSigningKey is not present generates signing keys', async function (assert) {
+    let keys = await generateAsymmetricKeys();
+    await service.importFromKey('name', keys.privateKey);
+    assert.notStrictEqual(await service.currentUser(), null);
+    assert.ok((await service.currentUser())?.publicSigningKey);
```

Comment:
the `currentUser` method loads the user if the user is not present. In this test, we expect to already have a user after the import, yeah? could we use the non-async `.record` instead?